llama-index-integrations/vector_stores/llama-index-vector-stores-milvus/llama_index/vector_stores/milvus/base.py

```diff
@@ -619,7 +619,7 @@ def query(self, query: VectorStoreQuery, **kwargs: Any) -> VectorStoreQueryResul
                 filter=string_expr,
                 limit=query.similarity_top_k,
                 output_fields=output_fields,
-                search_params=self.search_config,
+                search_params=kwargs["milvus_search_config"] if "milvus_search_config" in kwargs else self.search_config,
```

Comment:
You can do `search_params=kwargs.get("milvus_search_config", self.search_config)`

Reply:
I haven't looked at other vector db's and how they might handle (or whether they even support) these search parameters Since this was an adjustment only for the MilvusVectorStore I figured it would be in line with the "milvus_scalar_filters" "vector_store_kwargs" key to differentiate it as specifically Milvus.

Reply:
> kwargs.get("milvus_search_config", self.search_config)

You are right. I changed it to this to follow the pattern used in the rest of the class.